src/my-server.js

```diff
@@ -9,18 +14,26 @@ class Server {
 
     start () {
         return new Promise((resolve, reject) => {
-            this.server = this.app.listen(this.port, resolve).on('error', reject);
+            this.server = this.app.listen(this.port, resolve)
+                .once('error', err => {
+                    error(err);
+                    reject();
+                });
         })
-            .then(() => console.log(`Server started on ${this.host}:${this.port}`))
-            .catch(() => console.log(`Server failed to start`));
+            .then(() => log('Server started on %s:%d', this.host, this.port))
```

Comment:
Yeah, it's great that you know about formatting in the log function. But the template string looks simpler and more intuitive, isn't it?

Reply:
It is. But I thought that formatting will visually highlight params (in another color). But it doesn't look like that.